easybuild/easyblocks/generic/modulerc.py

```diff
@@ -58,9 +56,6 @@ def make_module_step(self, fake=False):
         modfile_path = self.module_generator.get_module_filepath(fake=fake)
         modulerc = os.path.join(os.path.dirname(modfile_path), self.module_generator.DOT_MODULERC)
 
-        if os.path.exists(modulerc) and not build_option('force'):
-            raise EasyBuildError("Found existing file at %s, not overwriting without --force", modulerc)
```

Comment:
This check is now being done in the `ModuleGenerator.modulerc` method (see https://github.com/easybuilders/easybuild-framework/pull/2611), so no point in doing it again here.